content/1_General/Adding_Solution.mdx

```diff
@@ -150,22 +131,50 @@ See [Working With MDX](/general/working-mdx) for additional information.
 
 Since code for the USACO Guide is contributed from several authors, code style
 will **not** be uniform. Of course, we still strive for code that is readable
-and understandable. If any code does not compile or is hard to read, please
-contact us.
-
-Please read the following resource before contributing code (although you do not
-have to follow all of the recommendations):
-
-<!--
-<Resource
-  source="MIT"
-  title="Course Notes: Style"
-  url="https://py.mit.edu/fall20/notes/style"
-  starred
->
-  Python-specific (but still generally useful)
-</Resource>
--->
+and understandable. If any code does not compile or is hard to read, contact us.
+
+Read our guidelines below before contributing code.
+
+### General
+
+- Use _tabs_ rather than spaces to indent.
+
+  <Info title="Pro Tip">
+  	If your code is indented with spaces, paste your code into the [Guide
+  	Editor](/editor), right-click, click "Command Palette" from the menu that
+  	pops up, and then search for "Convert Indentation to Tabs."
+  </Info>
+
+- Lines should not exceed 80 characters. If you need to, split comments across
+  multiple lines to follow this rule.
+- Add comments throughout your code to explain your logic. Feel free to
+  copy-paste the official USACO solution and improve it by adding more
+  descriptive variable names and helpful comments.
+- Be _consistent_ about formatting your code, unlike the following snippet. We
+  don't care about which one you use (`a+b` or `a + b`), but pick one and stick
+  with it throughout your entire code!
+
+```py
+ a,b = (0,1)
+ print(a+b)
+ print(a + b)
+```
+
+- Use universally understandable variable names, especially for Bronze and
+  Silver. Should be more descriptive than what you would normally use
+  in-contest.
+  - Do not name a data structure the data structure's name. For example, don't
+    name a `std::set` "set."
```

Comment:
Shouldn't this be "For C++, do not use names that could conflict with `using namespace std`. Therefore don't name `std::set` a set" or something like that? (At least, that was my intent when proposing this rule)

Reply:
I thought the intent was just to have names not be the structure's name in any language??